src/app/search/items/search-filters/known-values.ts

```diff
@@ -71,6 +71,26 @@ export const damageFilter = {
   fromItem: (item) => `is:${getItemDamageShortName(item)}`,
 } satisfies ItemFilterDefinition;
 
+const prismaticDamageLookupTable: LookupTable<string, string | undefined> = {
+  unknown: undefined,
+  arc: 'light',
+  solar: 'light',
+  void: 'light',
+  stasis: 'dark',
+  strand: 'dark',
+} as const;
+
+export const prismaticDamageFilter = {
+  keywords: ['light', 'dark'],
+  description: tl('Filter.PrismaticDamageType'),
+  filter:
+    ({ filterValue }) =>
+    (item) => {
+      const damageType = getItemDamageShortName(item) ?? 'unknown';
```

Comment:
I think you can use the damage type enum directly, so you don't need to go through a string unnecessarily.

Reply:
Good catch. I'm going to steal an idea from another damage type table and make the compiler check that all the enums are present, too.